src/engine/strat_engine/engine.rs

```diff
@@ -274,7 +278,12 @@ impl Engine for StratEngine {
 
             self.pools.insert(new_name.clone(), uuid, pool);
             devlinks::pool_renamed(&old_name, &new_name);
-            Ok(RenameAction::Renamed(uuid))
+            let result = RenameAction::Renamed(uuid);
+            info!(
+                "pool renamed: uuid {}, old name {}, new name {}",
+                result, old_name, new_name
+            );
+            Ok(result)
```

Comment:
At this point, this should be changed back to what was here before. We are now doing the logging at the D-Bus layer.

Reply:
Resolved.